### ISSUE TITLE
Error 10 since Capacitor 3

### ISSUE THREAD

**Issue by grallc** [CodetrixStudio/CapacitorGoogleAuth#154]
Hello,
My Google Auth workflow has been working perfectly for months, but now is broken since upgrading to Capacitor v3.
I've been across all the issues here but can't find a solution.
I'm basically facing an Error 10 whenever I try to login on Android (only, working perfectly on web and iOS).

I've checked all the keys, but can't find where it's not working.

Here is my signingReport:
![image](https://user-images.githubusercontent.com/14354161/137026070-017272de-8eab-407a-b4ee-47937dceecda.png)

And my firebase settings:
![image](https://user-images.githubusercontent.com/14354161/137026235-9667c543-9277-40b9-bc98-d240b0511075.png)

As well as my google cloud platform API Key setting:
![image](https://user-images.githubusercontent.com/14354161/137026313-67a89fb1-5957-4fa1-86f0-bd67f003d84d.png)

As you can see, the key is the same everywhere.
I've re-download the google-services.json file multiple times, but still not working.

Could you please help me ?

It's quite urgent since the issue is critical.

Thanks!

**Comment by grallc** [comment]
Another try :
Someone (#110 / @Read1337) has recommended me to try adding the Google Play SHA1 Key on the Firebase.
Unfortunately doesn't work neither:
![image](https://user-images.githubusercontent.com/14354161/137159909-ced6ebd7-a373-4e4e-a9ab-57b71db0e394.png)
![image](https://user-images.githubusercontent.com/14354161/137160062-2f898081-60cd-480f-ac29-e605497a00cf.png)


Still the same error

**Comment by DharmikJP** [comment]
Issue with release only?

I was struggling with same issue although I did all the configurations.
In my case google is managing my app signing key.
So initially when i deployed my app to app store, google sign in was not working and I got error 10.
Then I copied the SHA-1 from App Integrity and added in firebase project settings.

Initially I was trying to generate release apk directly from my Android Studio and test but it was failing all the time,
as my app was not signed with the google managed key.
So, I need to again generate android bundles and upload to play store.
And from App bundle explorer I downloaded signed ,universal apk and its working.
